conu/apidefs/container.py

```diff
@@ -203,7 +203,7 @@ def create(cls, image, container_params):
 
     def start(self):
         """
-        start current container
+        start current container - the container has to be running
```

Comment:
has to be created, not running (you shouldn't work so late :D)

Reply:
Oh my, you're right! :D